cfgov/v1/tests/test_password_policy.py

```diff
@@ -53,6 +55,11 @@ class TestMinimumPasswordAge(TestWithUser):
 
     def test_too_soon(self):
         user = self.get_user()
+        history_item = user.passwordhistoryitem_set.latest()
+        current_locked_until = history_item.locked_until
+        new_locked_until = current_locked_until.replace(year=datetime.MAXYEAR)
+        history_item.locked_until = new_locked_until
+        history_item.save()
         with self.assertRaises(ValidationError):
```

Comment:
I think that this test, as is, lacks enough context to be meaningfully testing anything. We're now setting it up so that the user is guaranteed to be locked, and all `can_change_password` does is to check if something is locked or not.  I think we need to be more explicit about what "too soon" means. This probably means a test for a new user, a test for an existing user that recently changed their password, etc. I can take those on, but do you agree this isn't actually testing anything as is?